src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js

```diff
@@ -0,0 +1,876 @@
+import { Validators } from 'react-reactive-form';
+
+import FormComponents from '../../components/form';
+import IncidentNavigation from '../../components/IncidentNavigation';
+
+export default {
+  controls: {
+    custom_text: {
+      meta: {
+        label: 'Dit hebt u net ingevuld:',
+        type: 'citation',
+        value: '{incident.description}',
+        ignoreVisibility: true,
+      },
+      render: FormComponents.PlainText,
+    },
+
+
+
+    // woningdelen
+    extra_wonen_woningdelen_vermoeden: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wat zich in deze woning afspeelt?',
+        subtitle: 'Vermoedens over bijvoorbeeld illegale activiteiten',
+        shortLabel: 'Vermoeden',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
```

Comment:
Are these values even necessary if, in every prop of `controls`, the value is the same?

Reply:
yes: pathMerge bepaald dat het een uitvraag pagina is. className gaat eruit als we

Reply:
My point is, the value for the `className` prop is the same everywhere. Can't we take it out of all of the config entries and move it to where the prop value is actually used?

Reply:
no that is only in the uitvraag pagina. there are questions with 100% width. I could make 'col-sm-12 col-md-8' the default value. current default is: col-12